app/server/utils/poli_payments.py

```diff
@@ -0,0 +1,106 @@
+import random, string, datetime, requests, base64
+from flask import current_app
+
+
+class PoliPaymentsError(Exception):
+    pass
+
+
+def generate_poli_host(path=None):
```

Comment:
as per owen's notes, perhaps `generate_poli_endpoint` given it extends the host?

Reply:
or `generate_complete_poli_path`?